app/src/main/java/com/example/cs65_final_project/spoonacular/SpoonacularGatewayController.java

```diff
@@ -0,0 +1,44 @@
+package com.example.cs65_final_project.spoonacular;
+
+import com.example.cs65_final_project.Ingredient;
+import com.example.cs65_final_project.Recipe;
+import com.example.cs65_final_project.exceptions.SpoonacularException;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Controller acting as a gateway to the Spoonacular client.
+ */
+public class SpoonacularGatewayController {
```

Comment:
@SoAmano Figured this class could be the bridge between the rest of the code and Spoonacular. Let me know what you think!